src/App.js

```diff
@@ -0,0 +1,18 @@
+import "./App.css";
+import Navbar from "./components/Navbar/index";
+import Main from "./components/Main/index";
+import Side from "./components/Side/index";
```

Comment:
We don't need to put index in import 
`import Navbar from "./components/Navbar";`